src/redis.ts

```diff
@@ -1,10 +1,29 @@
-import asyncRedis from "async-redis";
-import redis from "redis"
+import Redis from "ioredis"
 
-export const redisClient = redis.createClient(process.env.REDIS_PORT, process.env.REDIS_IP)
-let clientAsync 
+let connectionObj = {}, natMap = {}
 
-export const getAsyncRedisClient = () => {
-  clientAsync = clientAsync ?? asyncRedis.decorate(redisClient);
-  return clientAsync
+if(process.env.LOCAL === 'true') {
+  const REDIS_0_INTERNAL_IP = `${process.env.REDIS_0_INTERNAL_IP}:6379` 
+  const REDIS_1_INTERNAL_IP = `${process.env.REDIS_1_INTERNAL_IP}:6379`
+  const REDIS_2_INTERNAL_IP = `${process.env.REDIS_2_INTERNAL_IP}:6379`
+  
+  natMap = {
+    [REDIS_0_INTERNAL_IP]: { host: process.env.REDIS_0_DNS, port: process.env.REDIS_0_PORT },
```

Comment:
we should pass the default port if nothing is passed in the `REDIS_0_PORT` as a env variable

Reply:
This branch is for local development. The default port will probably never work with a minikube loadbalancer because the port range starts at `30000` and the default port is `26379`